### PR TITLE
Adjusting sync time

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2242

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/set-up-modern-hybrid-public-folders.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ Set-OrganizationConfig -PublicFoldersEnabled Remote -RemotePublicFolderMailboxes
 ```
 
 > [!NOTE]
-> You must wait until ActiveDirectory synchronization has completed to see the changes. This process can take up to 3 hours to complete. If you don't want to wait for the recurring synchronizations that occur every three hours, you can force directory synchronization at any time. For detailed steps to do force directory synchronization, see [Azure AD Connect sync: Scheduler](https://docs.microsoft.com/azure/active-directory/hybrid/how-to-connect-sync-feature-scheduler).
+> You must wait until ActiveDirectory synchronization has completed to see the changes. This process can take up to 30 minutes (by default) to complete. If you don't want to wait for the recurring synchronizations that occurs every 30 minutes, you can force directory synchronization at any time. For detailed steps to do force directory synchronization, see [Azure AD Connect sync: Scheduler](https://docs.microsoft.com/azure/active-directory/hybrid/how-to-connect-sync-feature-scheduler).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/set-up-modern-hybrid-public-folders.md:93]:
```suggestion
> You must wait until ActiveDirectory synchronization has completed to see the changes. This process can take up to 30 minutes (by default) to complete. If you don't want to wait for the recurring synchronizations that occur every 30 minutes, you can force directory synchronization at any time. For detailed steps to do force directory synchronization, see [Azure AD Connect sync: Scheduler](https://docs.microsoft.com/azure/active-directory/hybrid/how-to-connect-sync-feature-scheduler).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**